Libraries/CMSIS/Device/Maxim/MAX32662/Include/ptg_regs.h

```diff
@@ -151,43 +155,43 @@ typedef struct {
 
 /**
  * @ingroup  ptg_registers
- * @defgroup PTG_INTFL PTG_INTFL
- * @brief    Pulse Train Interrupt Flags
+ * @defgroup PTG_STOP_INTFL PTG_STOP_INTFL
+ * @brief    Pulse Train Stop Interrupt Flags
  * @{
  */
-#define MXC_F_PTG_INTFL_PT0_POS                        0 /**< INTFL_PT0 Position */
```

Comment:
Same comments as other reg files - redirect these but only if the generator can do it.